pkgs/development/tools/godot/mono/export-templates.nix

```diff
@@ -0,0 +1,25 @@
+{ godot-mono, lib }:
+
+# https://docs.godotengine.org/en/stable/development/compiling/compiling_with_mono.html#export-templates
```

Comment:
This should be in meta.homepage.

Reply:
Are you sure?  The URL is just what provides instructions for how to generate Godot export templates.  Export templates don't really have a "homepage".  Hence the inheritance of the base derivation's meta.homepage, which is just "https://godotengine.org".

Reply:
Linking to godotengine.org is very generic and linking to specific page for the export-template wouldn't hurt.